v63005/inref/gtm5574.m

```diff
@@ -26,13 +26,32 @@
 	read noct
 	use $p
 	close o
-	set fourgig=4294967296
+	if $ZCMDLINE<0 do neg
+	else  do pos
+	quit
+
+neg
+	set %DH=n
+	set %DO=n
+	set %DL=1
```

Comment:
Can you set %DL to a random value between 1 and 16 and randomly unset it too. All of them should continue to pass the existing tests. This way we test more cases.